pkg/fleet/env/env_test.go

```diff
@@ -200,11 +234,17 @@ func TestToEnv(t *testing.T) {
 				"DD_REMOTE_POLICIES=true",
 				"DD_INSTALLER_REGISTRY_URL=registry.example.com",
 				"DD_INSTALLER_REGISTRY_AUTH=auth",
+				"DD_INSTALLER_REGISTRY_USERNAME=username",
+				"DD_INSTALLER_REGISTRY_PASSWORD=password",
 				"DD_APM_INSTRUMENTATION_LIBRARIES=dotnet:latest,java,ruby:1.2",
 				"DD_INSTALLER_REGISTRY_URL_IMAGE=another.registry.example.com",
 				"DD_INSTALLER_REGISTRY_URL_ANOTHER_IMAGE=yet.another.registry.example.com",
 				"DD_INSTALLER_REGISTRY_AUTH_IMAGE=another.auth",
 				"DD_INSTALLER_REGISTRY_AUTH_ANOTHER_IMAGE=yet.another.auth",
+				"DD_INSTALLER_REGISTRY_USERNAME_IMAGE=another.username",
+				"DD_INSTALLER_REGISTRY_USERNAME_ANOTHER_IMAGE=yet.another.username",
+				"DD_INSTALLER_REGISTRY_PASSWORD_IMAGE=another.password",
+				"DD_INSTALLER_REGISTRY_PASSWORD_ANOTHER_IMAGE=yet.another.password",
```

Comment:
I find a bit confusing how is the customer or us able to leverage this ENV variables. 


Would you be able to provide an example?

Reply:
I assume you are talking about the image-specific ones: this is mostly for our internal testing because we have complex scenarios where we need to download the agent and installer package from the testing registry and the tracing packages from the prod registry for example. Customers probably will never need this.

Reply:
Thanks for the clarification